staging/src/k8s.io/apimachinery/pkg/util/proxy/transport_test.go

```diff
@@ -274,3 +277,91 @@ func TestProxyTransport(t *testing.T) {
 		testItem(name, &item)
 	}
 }
+
+func TestRewriteResponse(t *testing.T) {
+	gzipbuf := bytes.NewBuffer(nil)
+	flatebuf := bytes.NewBuffer(nil)
+
+	testTransport := &Transport{
+		Scheme:      "http",
+		Host:        "foo.com",
+		PathPrepend: "/proxy/node/node1:10250",
+	}
+
+	test := []struct {
+		encodeType string
+		excepted   []string
+		writer     func(string) *http.Response
+		reader     func(*http.Response) string
+	}{
+		{
+			encodeType: "gzip",
+			excepted: []string{
```

Comment:
1) I think you mean "expected" :)
2) This can be defined outside of the table and doesn't need to be repeated / different for each encoding type.

Reply:
Oops, I didn't notice that. Updated, thanks for the reminder.